<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1495</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tracking System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Video">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1300</width>
      <height>771</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Video</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>1280</width>
       <height>720</height>
      </rect>
     </property>
     <property name="text">
      <string>no signal</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="sources">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>uav1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>uav2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>uav3</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="conf_thres">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>110</y>
      <width>16</width>
      <height>621</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
    <property name="sliderPosition">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="nms_thres">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>110</y>
      <width>20</width>
      <height>621</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>45</number>
    </property>
    <property name="sliderPosition">
     <number>45</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="conf_show">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{background-color: rgb(255,255,255);}</string>
    </property>
    <property name="text">
     <string>CONF</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="nms_show">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{background-color: rgb(255,255,255);}</string>
    </property>
    <property name="text">
     <string>NMS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>740</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Confidence</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>740</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>NMS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
